src/frameworks/angular/index.ts

```diff
@@ -20,7 +20,11 @@ export const name = "Angular";
 export const support = SupportLevel.Experimental;
 export const type = FrameworkType.Framework;
 
-const CLI_COMMAND = join("node_modules", ".bin", process.platform === "win32" ? "ng.cmd" : "ng");
+const CLI_COMMAND = join(
+  execSync("npm root").toString().trim(),
```

Comment:
additional thought, we should make sure this is relative to the source directory

we should write it as a function: 

```ts
function getCLI(cwd: string) {
  return join(
    execSync("npm root", { cwd }),
    ...
```